update.md

```diff
@@ -0,0 +1,54 @@
+# ESI demo environment setup
+## Requirements
+* CentOS 8
+* Ussuri tripleo-repos
+## Steps
+* Download and install the python-tripleo-repos RPM
+```
+sudo dnf install -y https://trunk.rdoproject.org/centos8/component/tripleo/current/python3-tripleo-repos-0.1.1-0.20200612051255.ecf6206.el8.noarch.rpm
+```
+* Enable the current Ussuri repositories
+```
+sudo -E tripleo-repos -b ussuri current
+```
+* Install the TripleO CLI
+```
+sudo yum install -y python3-tripleoclient
+```
+* git clone esi-sandbox repo and cd esi-sandbox
+* Open deploy.sh, change INTERFACE=eth0 to INTERFACE=ens10; comment out ceph-ansible related commands.
+* Open standalone_parameters.yaml
+    * remove resource_registry: DockerRegistry, MistralEngine, MistralApi, MistralExecutor, MistralEventEngine
```

Comment:
I would just remove these from standalone_parameters.yaml and not even mention them here.

Reply:
An update: after I removed MistralEngine, MistralApi, MistralExecutor, MistralEventEngine and redeploy, the deployment failed. When I added them back, it succeeded again. I'm not sure if Mistral should be removed in this case.

Reply:
Oh, interesting - thanks for testing! What error did you get?

Reply:
The error was Container(s) with bad ExitCode: [''haproxy_restart_bundle'']

Reply:
Okay, that's really weird then. Is it possible that mistral was still referenced somewhere else - maybe in deploy.sh?

Reply:
deploy.sh has already removed mistral. I find that patches/docker directory references mistral, but not sure if they are used or not.

Reply:
Ah, they're not. Maybe see if it can be removed?

Reply:
After removing mistral related files in patches/docker, the same error still happened.